lib/plugins/aws/invoke/index.js

```diff
@@ -119,6 +118,10 @@ class AwsInvoke {
       logResult.split('\n').forEach(line => this.consoleLog(formatLambdaLogEvent(line)));
     }
 
+    if (invocationReply.FunctionError) {
```

Comment:
Do we need to add smth. like this in `invokeLocal` as well?

Don't have deep knowledge about implementation details here...